app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const {getTopics}= require("./controllers/topic-controllers")
+
+const app = express();
+app.use(express.json());
```

Comment:
We should not have this line of code, let's remove it and we can put it back when our request will have a body 🙂